.github/workflows/translations-pr.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - name: Run `npx lint:md --fix`
         # This runs a specific version of ESLint with only the Translation Pages Globbing
         # This avoid that unrelated changes get linted/modified within this PR
-        run: npx eslint "apps/site/pages/**/*.md?(x)" --fix --cache --cache-strategy=metadata --cache-file=apps/site/.eslintmdcache
+        run: npx eslint "apps/site/pages/**/*.md?(x)" --fix --cache --cache-strategy=metadata --cache-file=apps/site/.eslintmdcache --config=apps/site/eslint.config.js
```

Comment:
this should resolvehttps://github.com/nodejs/nodejs.org/actions/runs/10850360175/job/30111620392 post-merge

Reply:
when the proper review time passes, I will likely admin merge this and then kick off another sync. the alternative was to contribute the fix separately and wait, OR hotfix it onto main. any which way works, as long as we are committed to monitoring it

Reply:
Can you make a dedicated PR for the fix, because this is wrong actually, I can comment and review on a dedicated PR :)

Reply:
@ovflowd take a look at https://github.com/nodejs/nodejs.org/pull/7043